public/app/features/admin/ldap/LdapDrawer.tsx

```diff
@@ -233,15 +263,151 @@ export const LdapDrawerComponent = ({ onClose }: Props) => {
                   'ldap-drawer.extra-security-section.tls-ciphers.placeholder',
                   'e.g. ["TLS_AES_256_GCM_SHA384"]'
                 )}
-                value={watch('settings.config.servers.0.tls_ciphers')}
+                value={watch(`${serverConfig}.tls_ciphers`) || ''}
                 onChange={({ currentTarget: { value } }) =>
                   setValue(
-                    'settings.config.servers.0.tls_ciphers',
+                    `${serverConfig}.tls_ciphers`,
                     value?.split(/,|\s/).map((v: string) => v.trim())
                   )
                 }
               />
             </Field>
+            <Field
+              label={t(
+                'ldap-drawer.extra-security-section.encryption-provider.label',
+                'Encryption key and certificate provision specification (required)'
+              )}
+              description={t(
+                'ldap-drawer.extra-security-section.encryption-provider.description',
+                'X.509 certificate provides the public part, while the private key issued in a PKCS#8 format provides the private part of the asymmetric encryption.'
+              )}
+            >
+              <RadioButtonGroup
+                id="encryption-provider"
+                options={[
+                  {
+                    label: t(
+                      'ldap-drawer.extra-security-section.encryption-provider.base-64',
+                      'Base64-encoded content'
+                    ),
+                    value: EncryptionProvider.Base64,
+                  },
+                  {
+                    label: t('ldap-drawer.extra-security-section.encryption-provider.file-path', 'Path to files'),
+                    value: EncryptionProvider.FilePath,
+                  },
+                ]}
+                value={encryptionProvider}
+                onChange={setEncryptionProvider}
+              />
+            </Field>
+            {encryptionProvider === EncryptionProvider.Base64 && (
+              <>
+                <Field
+                  label={t(
+                    'ldap-drawer.extra-security-section.root-ca-cert-value.label',
```

Comment:
All the translation keys for this page [should be three levels](https://github.com/grafana/grafana/blob/main/contribute/internationalization.md#phrase-id-naming-convention). Maybe you could replace the last `.` with a `-` instead?

Reply:
This is going to affect all of the translations. I'll merge the first 2 keys in all the instances. This is consistent with the LdapSettingsPage levels for page.item.labels, page.item.description, page.item.placeholder

Reply:
changed as requested